.buildkite/pipeline.yml

```
@@ -501,6 +523,7 @@ steps:
           --moist equil
           --job_id edmf_soares
           --dt 0.25secs
+          - julia --color=yes --project=examples regression_tests/test_mse.jl --job_id edmf_soares --out_dir edmf_soares
```

```suggestion
        command: julia --color=yes --project=examples regression_tests/test_mse.jl --job_id edmf_soares --out_dir edmf_soares
```
What if we did this instead?